### PR TITLE
[DemoApp] Fix deployment with netlify

### PR DESCRIPTION
**Pull request by dtassone**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.54",
     "@material-ui/monorepo": "https://github.com/mui-org/material-ui.git#next",
+    "netlify-plugin-cache-docs": "https://github.com/mui-org/material-ui.git#next",
```
Comment (oliviertassinari) [inline package.json:47]: We need to use the plugin from the main repository.

**Review by oliviertassinari — CHANGES_REQUESTED** [review]
The deployment of "material-ui-x" app is failing. The deployment of muix-preview doesn't seem to trigger for each pull request: https://app.netlify.com/sites/muix-preview/overview. 

I would suggest we

- drop "muix-preview" as Netlify doesn't seem to support two apps on the same repository
- we either build demo-app inside "material-ui-x" like we do for the storybook or we move the demo-app as a new Next.js page.

**Comment by dtassone** [comment]
@material-ui/monorepo => is

> It's not working, I would suggest we simply move the demo app as a new page for Next.js.

We need to exclude the demo-app node_modules from the tests
Or use only the required package, 'netlify-plugin-cache-docs' for deployment, refactor the taml file to point to this one directly
I don't understand why we need the full mono-repo dependency which brings a lot of packages that we don't need

**Comment by oliviertassinari** [comment]
I have updated my previous comment. I think that the first problem to fix is having two Netlify apps instead of one. 

> I don't understand why we need the full mono-repo dependency which brings a lot of packages that we don't nee

It's important so we operate as if we have the same repository.

**Comment by dtassone** [comment]
> The deployment of "material-ui-x" app is failing. The deployment of muix-preview doesn't seem to trigger for each pull request: https://app.netlify.com/sites/muix-preview/overview.
> 
> I would suggest we
> 
> * drop "muix-preview" as Netlify doesn't seem to support two apps on the same repository
> * we either build demo-app inside "material-ui-x" like we do for the storybook or we move the demo-app as a new Next.js page.

> we either build demo-app inside "material-ui-x" like we do for the storybook
I suggest we do that

**Pull request closed without merging**